src/routes/safe/components/Balances/dataFetcher.ts

```diff
@@ -13,6 +13,25 @@ export const BALANCE_TABLE_VALUE_ID = 'value'
 
 const { nativeCoin } = getNetworkInfo()
 
+const getTokenPriceValue = (token: Token, currencyValues?: BalanceCurrencyList, currencyRate?: number): string => {
```

Comment:
this whole function only difers from `getTokenPriceInCurrency` by the returned value.

WDYT about extracting the common logic to another function that will return the `balance` instead?

BTW, wouldn't it be better if it's not fixed to `2` decimals the returned value for this field?